packages/runtime/src/components/builtin/Text/ReadOnlyText.tsx

```diff
@@ -84,6 +84,30 @@ function Document({ nodes }: DocumentJSON) {
   )
 }
 
+type OrderedListProps = ComponentPropsWithoutRef<'ol'>
+
+function OrderedList(props: OrderedListProps) {
+  return <ol className="ordered-list" {...props} style={{ listStylePosition: 'inside' }} />
```

Comment:
I'm guessing there's no need to spread the style and className props?

Reply:
I normally would, but in this case I copied the implementation from `@convertkit/slate-lists` since we're going for as much parity as possible